daemon/network.go

```diff
@@ -306,6 +313,29 @@ func (daemon *Daemon) createNetwork(create types.NetworkCreateRequest, id string
 	}, nil
 }
 
+func (daemon *Daemon) pluginRefCount(driver, capability string, mode int) {
+	var builtinDrivers []string
+
+	if capability == driverapi.NetworkPluginEndpointType {
+		builtinDrivers = daemon.netController.BuiltinDrivers()
+	} else if capability == ipamapi.PluginEndpointType {
+		builtinDrivers = daemon.netController.BuiltinIPAMDrivers()
+	}
+
+	for _, d := range builtinDrivers {
+		if d == driver {
+			return
+		}
+	}
+
+	if daemon.PluginStore != nil {
+		_, err := daemon.PluginStore.Get(driver, capability, mode)
+		if err != nil {
+			logrus.Errorf("Error handling plugin refcount operation for mode=%d driver=%s", mode, driver)
```

Comment:
I'd suggest making this `logrus.WithError(err).WithFields(logrus.Fields{"mode": mode, "driver", driver}).Error("Error handling plugin refcount operation")`

That way the error will not be hidden, and the mode and driver information will be captured in a more structured way.